packages/transformer/src/test/standalone/IModelTransformerHub.test.ts

```diff
@@ -857,5 +857,75 @@ describe("IModelTransformerHub", () => {
     await tearDown();
     sinon.restore();
   });
+
+  it("should delete definition elements and models when processing changes", async () => {
+    let definitionPartitionId1: string;
+    let definitionModelId1: string;
+    let definitionContainerId1: string;
+    let definitionModelId2: string;
+
+    const timeline: Timeline = {
```

Comment:
nit: can use an array here if you prefer. I only used an explicitly integer-indexed object in an older version of this test utility but it's still usable so not necessary to change

Reply:
Since it's still usable, I will leave it like it is